lib/improver/wind_direction.py

```diff
@@ -163,34 +196,58 @@ def complex_to_deg(complex_in):
 
         return angle
 
-    @staticmethod
-    def find_r_values(complex_in):
+    def wind_dir_mean(self):
+        """Find the mean wind direction using complex average which actually
+           signifies a point between all of the data points in POLAR
+           coordinates - NOT the average DEGREE ANGLE.
+
+        Uses:
+            self.wdir_complex (np.ndarray or float):
+                3D array or float - wind direction angles in degrees.
+            self.realization_axis (int):
+                Axis to collapse over.
+
+        Defines:
+            self.wdir_mean_complex (np.ndarray or float):
+                3D array or float - wind direction angles as complex numbers
+                collapsed along an axis using np.mean().
+            self.wdir_slice_mean (np.ndarray or float):
+                3D array or float - wind direction angles in degrees collapsed
+                along an axis using np.mean().
+        """
+        self.wdir_mean_complex = np.mean(self.wdir_complex,
+                                         axis=self.realization_axis)
+        self.wdir_slice_mean.data = self.complex_to_deg(self.wdir_mean_complex)
+
+    def find_r_values(self):
         """Find radius values from complex numbers.
 
         Takes input wind direction in complex values and returns array
         containing r values using Pythagoras theorem.
 
-        Args:
-            complex_in (np.ndarray or float):
+        Uses:
+            self.wdir_mean_complex (np.ndarray or float):
+                3D array or float - wind direction angles in complex numbers.
```

Comment:
This has the same description as the variable below. Could you change the description to something like:
```
 self.wdir_slice_mean (iris.cube.Cube):
 "3D array or float - average wind direction angles in complex numbers."
```